extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt

```diff
@@ -81,6 +87,14 @@ Modifications to the EGL 1.5 Specification
     Screen Rendering Surfaces, before "The default value of EGL_GL_COLORSPACE
     is EGL_GL_COLORSPACE_LINEAR.":
 
+    [[ If EGL_EXT_gl_colorspace_bt2020_hlg is supported ]]
+
+    If its value is EGL_GL_COLORSPACE_BT2020_HLG_EXT, then a non-linear, HLG
+    encoded BT.2020 color space is assumed, with a corresponding GL_FRAMEBUFFER-
+    _ATTACHMENT_COLOR_ENCODING value of GL_LINEAR, as neither OpenGL nor OpenGL
+    ES supports HLG framebuffers. Applications utilizing this option need to
+    ensure that HLG encoding is performed on the application side.
```

Comment:
Does expanding the acronym add any value?:
"HLG (Hybrid Log Gamma) ..."

Reply:
Done.